virtual-organizations/ARA.yaml

```diff
@@ -22,6 +22,11 @@ ID: 138
 LongName: Askaryan Radio Array
 Name: ARA
 OASIS:
+  Managers:
+    David Schultz:
+      DNs:
+      - /DC=org/DC=cilogon/C=US/O=University of Wisconsin-Madison/CN=DAVID SCHULTZ A11726
+      ID: 67523ec91c983a4c0c1c576beeac1037aa863da3
```

Comment:
```suggestion
      ID: b512ff43d2e9239b62fb63bf3b30202fa30731e9
```

Reply:
Ah, I copy/pasted the wrong one. Thanks.